lang/python/python-jsonschema/Makefile

```
@@ -0,0 +1,43 @@
+#
+# Copyright (C) 2020 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-jsonschema
+PKG_VERSION:=3.2.0
+PKG_RELEASE:=1
+
+PYPI_NAME:=jsonschema
+PKG_HASH:=c8a85b28d377cc7737e46e2d9f2b4f44ee3c0e1deac6bf46ddefc7187d30797a
+
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=COPYING
+PKG_MAINTAINER:=Javier Marcet <javier@marcet.info>
+
+include ../pypi.mk
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+PYTHON3_PKG_SETUP_ARGS:=
+
+define Package/python3-jsonschema
+  SECTION:=lang
+  CATEGORY:=Languages
+  SUBMENU:=Python
+  TITLE:=An implementation of JSON Schema validation for Python
+  URL:=https://github.com/Julian/jsonschema
+  DEPENDS:=+PACKAGE_python3-jsonschema:python3-light
```

```suggestion
  DEPENDS:=+python3-light +python3-attrs +python3-six +python3-pyrsistent
```

See https://github.com/Julian/jsonschema/blob/master/setup.cfg#L33